crates/ruff/src/checkers/imports.rs

```diff
@@ -54,5 +54,57 @@ pub fn check_imports(
             &blocks, python_ast, locator, stylist, settings, autofix,
         ));
     }
-    diagnostics
+    let mut imports = Imports::default();
+    let mut imports_vec = vec![];
+    // find the difference between the current path and the package root (inc root)
+    let modules: Vec<&str> = match package {
+        Some(package) => {
+            let mut modules: Vec<&str> = vec![package.iter().last().unwrap().to_str().unwrap()];
+            modules.extend(
+                path.strip_prefix(package)
+                    .iter()
+                    .rev()
+                    // we don't want the end module as it is the current one
+                    .skip(1)
+                    .map(|p| p.to_str().unwrap())
+                    .rev(),
+            );
+            modules
+        }
+        None => path
+            .iter()
+            .rev()
+            .skip(1)
+            .take(1)
+            .map(|p| p.to_str().unwrap())
+            .collect::<Vec<_>>(),
+    };
```

Comment:
Can we reuse `crates/ruff_python_ast/src/helpers.rs#to_module_path` for this?

Reply:
I'll look into it.

Reply:
My only qualm with using it is that it returns me a `Vec<String>` whereas here we're using `Vec<&str>`